jenkins/deployment/community/cortx-community-deploy.groovy

```diff
@@ -15,156 +14,168 @@ pipeline {
         ansiColor('xterm')
     }
 
-
     parameters {
         string(name: 'CORTX_RE_BRANCH', defaultValue: 'main', description: 'Branch or GitHash for CORTX Cluster scripts', trim: true)
-        string(name: 'CORTX_RE_REPO', defaultValue: 'https://github.com/Seagate/cortx-re/', description: 'Repository for CORTX Cluster scripts', trim: true)
+        string(name: 'CORTX_RE_REPO', defaultValue: 'https://github.com/Seagate/cortx-re', description: 'Repository for CORTX Cluster scripts', trim: true)
         string(name: 'CORTX_TAG', defaultValue: 'main', description: 'Branch or GitHash for generaing CORTX container images', trim: true)
```

Comment:
We need to keep the `CORTX_TAG` parameter. This is needed to test community deployment with custom tag e.g. `2.0.0.-940` instead of `main`

Reply:
CORTX_TAG parameter is already in the code

Reply:
@shailesh-vaidya, these parameters are there now as,
<img width="312" alt="image" src="https://user-images.githubusercontent.com/65269273/189631807-71e39943-e6c1-455b-9833-ff64ab09e78a.png">